web/src/layout/leftpanel/countrypanel.js

```diff
@@ -211,24 +212,24 @@ class Component extends React.PureComponent {
               <hr />
               <div className="country-history">
                 <div className="loading overlay" />
-                <span
-                  className="country-history-title"
-                  dangerouslySetInnerHTML={{ __html: co2Sub(__('country-history.carbonintensity24h')) }}
-                />
+                <span className="country-history-title">
+                  {co2Sub(__(
+                    tableDisplayEmissions
+                      ? 'country-history.emissions24h'
+                      : 'country-history.carbonintensity24h'
+                  ))}
+                </span>
                 <br />
                 <small className="small-screen-hidden">
                   <i className="material-icons" aria-hidden="true">file_download</i> <a href="https://data.electricitymap.org/?utm_source=electricitymap.org&utm_medium=referral&utm_campaign=country_panel" target="_blank">{__('country-history.Getdata')}</a>
                   <span className="pro"><i className="material-icons" aria-hidden="true">lock</i> pro</span>
                 </small>
 
-                <CountryHistoryCarbonGraph />
+                {tableDisplayEmissions ? <CountryHistoryEmissionsGraph /> : <CountryHistoryCarbonGraph />}
```

Comment:
I'm just thinking aloud here: the graphs won't animate using a transition when we toggle `tableDisplayEmissions`. That's a shame. Wouldn't it be simpler if we re-used the same elements, but just passed different data to it? That would achieve the animation effect.
Actually I didn't test it so I'm not sure about the statement I'm making.

Reply:
That's a good point, but graph animations are already not there as you can see by toggling `tableDisplayEmissions` and observing the mix graph (which passes two different datasets to one `AreaGraph`).

The animations were lost when we started rendering graphs with React instead of D3, but it wouldn't be too hard to reintroduce them with some combination of CSS and React animations (e.g. see https://github.com/tmrowco/electricitymap-contrib/issues/2286).

However, while having graph layers shrink and expand in a smooth motion make for a nice UX when the coordinate units are fixed, I think it might be misleading and undesirable when switching between completely different graphs. So I'd personally opt out of animations here even if there was no additional work involved :)